masschallenge/impact-api
# [AC-7542] Ability to filter /api/v1/user/ by date modified

[JIRA](https://masschallenge.atlassian.net/browse/AC-7542)

Turns out we already have this functionality, so this PR just adds a test to verify the overall functionality requested by the ticket indeed operated correctly 

To Test:
- navigate to `/api/v1/user`
- add `?updated_at.after=2016-04-23T18:25:43.511Z`(or any proper ISO 8601 date)
- * should only report users modified after this date
- replace the parameter with `?updated_at.before=2016-04-23T18:25:43.511Z`(or any proper ISO 8601 date)
- * should only report users modified before this date
- now, try adding `?updated_at.after=2016-04-23T18:25:43.511Z&?updated_at.before=2019-04-23T18:25:43.511Z`(or any two proper iso dates where the latter is more recent than former)
- * should only report users modified between these two dates
